OHIF/Viewers
# Fix/Suggestion(MeasurementTracking):prevent empty sr description when export

### Context
Suggestion to prevent empty description for DICOM SR Report export. Prevent and alert when users attempts to save with empty description.

Questions:
1. Alert currently using browser, would bringing UINotificationService be preferred?
2. Alternatively could use states to disable the save button; however, ran into issues with react hooks when implementation attempted to turn actions passed to Dialog component to state -> is there any examples of a different approach to this?



### Changes & Results
-use _handleFormSubmit when enter is pressed (so same code is ran)
-add empty description check before allow processing within _handleFormSubmit
-if empty, stop event resolution and send alert



### Testing
1. Load Image in Basic Viewer
3. Select length tool
4. Create length
5. Start tracking
6. Change to Measurements panel (on right panel)
7. Click "Create Report" button
8. Press enter with empty description
-> Should ignore and prevent save
9. Click save button with empty description
-> Should ignore and prevent save
10. Type description, then delete (so description is blank again)
11. Press enter with empty description
-> Should ignore and prevent save
12. Click save button with empty description
-> Should ignore and prevent save
13. Type description, click save/press enter
-> Should allow save routine to proceed


### Checklist

#### PR



  semantic-release format and guidelines.

#### Code

  etc.)

#### Public Documentation Updates



  additions or removals.

#### Tested Environment

- [x] "Node version: 16.15.1 
- [x] "Browser: Chrome 110.0.5481.96
  


[blog]: https://circleci.com/blog/triggering-trusted-ci-jobs-on-untrusted-forks/
[script]: https://github.com/jklukas/git-push-fork-to-upstream-branch
